schema/codelists/partyRole.csv

```diff
@@ -10,3 +10,9 @@ payee,Payee,A party in receipt of a payment from a transaction.,
 reviewBody,Review body,A party responsible for the review of this procurement process. This party often has a role in any challenges made to the contract award.,
 interestedParty,Interested party,"A party that has expressed an interest in the contracting process: for example, by purchasing tender documents or submitting clarification questions.",
 contractImplementationManager,Contract implementation manager,The organization responsible for managing the implementation of the contract on behalf of the buyer. (This may be different from the procuring entity that manages the contracting process.),
+mediationBody,Mediation body,The body responsible for mediation procedures.,
+centralPurchasingBody,Central purchasing body,"The procuring entity providing centralized purchasing activities and, possibly, ancillary purchasing activities.",
+processContactPoint,Process contact point,A contact point dedicated to this contracting process.,
+reviewContactPoint,Review contact point,The service from which information about the review procedure can be obtained.,
+selectedParticipant,Selected participant,A party that has already been selected to participate in the design contest.,
```

Comment:
This was added to the EU extension for IV.1.7 "Names of participants already selected (in the case of a restricted contest)" from the current EU forms.

@ColinMaudry Should this instead be modelled as a Bid with a status of 'invited'? https://extensions.open-contracting.org/en/extensions/bids/master/codelists/#bidStatus.csv

Reply:
Good idea, but I see several issues:

- That would be a bid with only `status` and `tenderers`, no `date` and no `value`.
- At the time of publication of this design contest notice, we don't know whether the selected/invited participant actually submitted a bid. They may be selected/invited but not participate. @JachymHercher Could you please confirm this point?
- Only the "selected participants" would appear in bids, as if they were the only bidders (F13 Results of design contest only discloses statistics about participants, not details)

Reply:
Yes, there are two issues: (1) how to model incomplete data, and (2) how to model invitations to bid.

(1) From this perspective, it's no problem for a bid to lack a `date` and `value`, and for `bids` to only contain invited applicants. A more complete dataset would add/update such values.
(2) However, maybe there should be no such status as 'invited', because an invitation to bid is not the same thing as a bid, and therefore they should be modelled separately.

We should move this to an issue to discuss.

Reply:
Actually, there is already an issue: https://github.com/open-contracting/ocds-extensions/issues/141

Reply:
For now 'selectedParticipant' remain in the EU extension.

Reply:
Done in https://github.com/open-contracting-extensions/ocds_eu_extension/pull/11